app.py

```diff
@@ -17,7 +17,7 @@
 
 import configuration
 from py_modules import backend_api
-from py_modules.db_api import DBApi
+from py_modules.db_api import DBApi, insert_custom_registration
```

Comment:
I don't recommend direct import of a function. It would be useful to insert a `DBHelper` class in db_api.py and put this new function inside DBHelper (great job!) In order to be able to keep the code clean and avoid other specific imports in the future